src/oref_alert_indexer.py

```diff
@@ -0,0 +1,70 @@
+import pandas as pd
+import matplotlib.pyplot as plt
+import warnings
+warnings.filterwarnings('ignore')
+import requests
+import json
+from datetime import datetime, timedelta
+import time 
+from errors import OrefAPIException
+import os
+from dotenv import load_dotenv, dotenv_values
+import logging
+
+load_dotenv()
+URL = os.getenv('ALERTS_URL')
+ALERTS_DATA_FILE = os.getenv('ALERTS_DATA_FILE')
+DAYS_INTERVAL = 2
+SLEEP_TIME = 5
+
+
+class OrefAlertsIndexer:
+    # retrieve the alerts data documentation from oref alerts api for which the alert category is missiles
+    def get_alerts_from_oref_api(self, current_date, dest_date):
```

Comment:
write a decorator that do retries for 3 time if this functin fails.  (sleeps 5 seconds between each try)
and if fails 3 times only - only then raise that error